lib/Doctrine/Migrations/Tools/Console/ConsoleRunner.php

```diff
@@ -99,7 +99,9 @@ public static function run(array $commands = [], ?DependencyFactory $dependencyF
     /** @param DoctrineCommand[] $commands */
     public static function createApplication(array $commands = [], ?DependencyFactory $dependencyFactory = null): Application
     {
-        $cli = new Application('Doctrine Migrations', Versions::getVersion('doctrine/migrations'));
+        $version = InstalledVersions::getVersion('doctrine/migrations');
+        assert($version !== null);
```

Comment:
`getVersion` returns `null` when the package name is provided by another package rather than being installed directly (if the package is not available in any way, an exception is thrown instead). But as we ask the version for our own package, we know that it is not provided by another package (otherwise, this LOC is not there either to execute it).